app/javascript/containers/BuildingListContainer.jsx

```diff
@@ -5,23 +5,30 @@ import { getBuildings } from '../selectors/buildingsSelector';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
+import BuildingInfoContainer, { } from './BuildingInfoContainer';
+import ContinueButtonContainer from './ContinueButtonContainer';
+import NotificationBarContainer from './NotificationBarContainer';
 
 class BuildingListContainer extends React.Component {
   render() {
     const buildings = this.props.buildings;
-    return (<div>
-      <h2>Buildings</h2>
+    return (<div className='building_list_container'>
+      <h1>Buildings</h1>
+      <div className='building_list'>
       <hr />
       {Object.keys(buildings).map(id => {
-        return (<p key={id}>{this.props.buildings[id].name} |
-          <Link to={`/buildings/${id}`}>Details</Link>
-        </p>)
+        return (<div key={id} className='building_view'>
+          <NotificationBarContainer building_id={id}></NotificationBarContainer>
+          <BuildingInfoContainer building_id={id} className='building_view_info'></BuildingInfoContainer>
+          <ContinueButtonContainer building_id={id}></ContinueButtonContainer>
+        </div>)
       })}
+      </div>
     </div>);
   }
 }
 
-function mapStateToProps(state) {
+function mapStateToProps(state, ownProps) {
```

Comment:
`ownProps` isn't necessary unless you need to refer to this component's props somewhere in `mapStateToProps` (like in the arguments to selector methods)